#!/usr/bin/env python

PACKAGE = 'coral_usb'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('score_thresh', double_t, 0, 'Threshold for confidence score', 0.60, 0.0, 1.0)
gen.add('top_k', int_t, 0, 'Threshold for confidence score', 100, 0, 1000)
gen.add('model_file', str_t, 0, 'Model file path', 'package://coral_usb/models/mobilenet_ssd_v2_coco_quant_postprocess_edgetpu.tflite')
gen.add('label_file', str_t, 0, 'Label file path', 'package://coral_usb/models/coco_labels.txt')

exit(gen.generate(PACKAGE, PACKAGE, 'EdgeTPUObjectDetector'))
